keyboards/ergodash/rev1/keymaps/shadowprogr/keymap.c

```diff
@@ -0,0 +1,206 @@
+#include QMK_KEYBOARD_H
+
+
+#define _WINDOWS 0
+#define _LINUX 1
+#define _NUMPAD 4
+#define _LOWER 8
+#define _RAISE 9
+#define _ADJUST 16
```

Comment:
Two things here: 
* We recommend using an Enum here, since that makes things easier to managed long term.  It handles the number incrementation, so you don't have to, as you have here. 
* By using these higher number, you're causing the firmware to auto-populate layers (2,3,5,6,7,10-15) with 0's, and artificially inflating the firmware size, for no gain.  

```suggestion
enum layers {
    _WINDOWS,
    _LINUX, 
    _NUMPAD,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```